COPYRIGHT.txt

```diff
@@ -228,11 +228,6 @@ Comment: WebP codec
 Copyright: 2010, Google Inc.
 License: BSD-3-clause
 
-Files: ./thirdparty/libwebsockets/
```

Comment:
You should add `./thirdparty/wslay/` to `COPYRIGHT.txt` too.

Reply:
done :+1: